### ISSUE TITLE
McAfee ATD Analyzer

### ISSUE THREAD

**Issue by bullerdude** [CERT-BDF/Cortex-Analyzers#25]
### Request Type
Analyzer Request

### Work Environment
N/A

### Problem Description
Create an analyzer that will submit files to a local McAfee ATD sandbox instance and retrieve the report and indicators that are generated

**Comment by saadkadhi** [comment]
Hi @bullerdude. As for FireEye AX, we do not have access to an instance hence if you or someone would be willing to create an analyzer for it or give us access to one... you may contact us on our support email address if you'd like to discuss this. Thanks.

**Comment by bullerdude** [comment]
Hi @saadkadhi, we are keen to develop this analyser. We will develop the code using a forked repository - https://github.com/UNIT777/Cortex-Analyzers.

**Comment by saadkadhi** [comment]
Thanks @bullerdude. Once your analyzer is ready, please submit a pull request so we integrate it to the analyzer list. Also, let us know if we can be of any help for coding the analyzer. 

Once it is dubbed ready for production, we will work on TheHive's short and long report templates. For that, we will need some sample outputs from the analyzer. Otherwise, if you feel confortable with AngularJS, please go ahead and provide short and long report templates so we can package them and share them with the community.

Thanks a heap for your help.

**Comment by saadkadhi** [comment]
Hi @bullerdude. Any update on this front? Thanks a lot.

**Comment by Vince147** [comment]
Hey @saadkadhi, i will check up on this. I was looking to build a tool to poke ATD and I discovered Cortex, which is what i wanted. I plan to do this this during this month. By that time, if @bullerduke have something, let us know.

**Comment by bullerdude** [comment]
Yes we have the integration working; it required a custom piece of middleware to work around session management issues in the ATD API.

Will try to get it uploaded during the week.

________________________________

**Comment by Edward-merrett** [comment]
Hey,

Is there an update on this? 

Would really love to integrate with ATD. 

Ed
